code-challenges.test.js

```diff
@@ -24,13 +24,30 @@
 // Example input: 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
-
+//use .length
 
 
 
 // b) Create the function that makes the test pass.
-
-
+describe("when a number greater than 2 is called", () => {
+  it("returns an array of the length containing the Fibonacci sequence", () => {
+    //arrange
+    const myArray = function (a)
+    {
+      if (a===1)
+      {
+        return [0,1];
+      } else {
+        var b = myArray(a-1);
+        b.push(b[b.length - 1] + b[b.length - 2]);
+        return b;
+      }
+    }
+expect(myArray).toEqual([])
```

Comment:
there should also be expectations for when a value like 6 or 10 is passed into the function